File: sandbox/dotnet/src/Dropbox.SignSandbox/AccountVerifyExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class AccountVerifyExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.Username = "YOUR_API_KEY";
        // config.AccessToken = "YOUR_ACCESS_TOKEN";

        var accountVerifyRequest = new AccountVerifyRequest(
            emailAddress: "some_user@dropboxsign.com"
        );

        try
        {
            var response = new AccountApi(config).AccountVerify(
                accountVerifyRequest: accountVerifyRequest
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling AccountApi#AccountVerify: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: examples/AccountVerifyExample.py
import json
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration(
    username="YOUR_API_KEY",
    # access_token="YOUR_ACCESS_TOKEN",
)

with ApiClient(configuration) as api_client:
    account_verify_request = models.AccountVerifyRequest(
        email_address="some_user@dropboxsign.com",
    )

    try:
        response = api.AccountApi(api_client).account_verify(
            account_verify_request=account_verify_request,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling AccountApi#account_verify: %s\n" % e)
